components/freshsales/actions/create-contact/create-contact.mjs

```
@@ -0,0 +1,106 @@
+          is_primary: true,
+        },
+      ];
+      parseObject(data.additionalAccounts)?.map((account) => {
+        data.sales_accounts.push(
+          {
+            id: account,
+            is_primary: false,
+          },
+        );
+      });
+      delete data.primaryAccount;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)